src/NzbDrone.Core/Extras/Metadata/Consumers/Wdtv/WdtvMetadata.cs

```diff
@@ -111,10 +111,10 @@ public override MetadataFileResult TrackMetadata(Artist artist, TrackFile trackF
                     var details = new XElement("details");
                     details.Add(new XElement("id", artist.Id));
                     details.Add(new XElement("title", string.Format("{0} - {1} - {2}", artist.Name, track.TrackNumber, track.Title)));
-                    details.Add(new XElement("artist_name", artist.Name));
+                    details.Add(new XElement("artist_Name", artist.Metadata.Value.Name));
```

Comment:
Did you mean to change case on `artist_Name`

Reply:
Oops...